lib/gst/player/gstplayer.c

```diff
@@ -3179,6 +3180,84 @@ gst_player_set_visualization_enabled (GstPlayer * self, gboolean enabled)
       enabled ? "Enabled" : "Disabled");
 }
 
+/**
+ * gst_player_get_color_balance_list_channels:
+ * @player: #GstPlayer instance
+ *
+ * Retrieve a list of the available channels.
+ *
+ * Returns: (element-type Gst.ColorBalanceChannel) (transfer none): A
+ *          GList containing pointers to #GstColorBalanceChannel
+ *          objects. The list is owned by the #GstColorBalance
+ *          instance and must not be freed.
```

Comment:
This would mean that the list is owned by playbin, and as such is valid as long as the GstPlayer instance is? Or would it suddenly become invalid if e.g. the video sink changes, we switch tracks, etc.?

If it can become suddenly invalid, please make this (transfer full) by copying the list and taking another reference of each channel.